lib/ImportStatement.js

```diff
@@ -148,11 +148,8 @@ export default class ImportStatement {
   /**
    * @return {Array} an array that can be used in `sort` and `uniq`
    */
-  toNormalized(): Array<string> {
-    if (!this.defaultImport && !this.hasNamedImports() && this.hasSideEffects) {
-      return [this.path];
-    }
-    return [this.defaultImport || '', ...this.localNames()];
+  toNormalized(): Array<any> {
+    return [this.defaultImport !== 'React', this.path];
```

Comment:
Can you help me understand why this change is made (including the type change)? It seems weird to have React hard-coded in this project.

Reply:
I reverted the change. Now it is only [this.path]